test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -316,55 +315,86 @@ define([
             }
         }
 
+        // is the error marker showing?
```

Comment:
The majority of the changes to this file are either editing variable names for clarity or adding in some extra checks in preparation for error responses.